blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarterTest.java

```diff
@@ -0,0 +1,275 @@
+package com.rackspacecloud.blueflood.service;
+
+import static junit.framework.Assert.*;
+
+import org.junit.*;
+
+import java.io.IOException;
+
+public class BluefloodServiceStarterTest {
+
+    @Before
+    public void setUp() throws IOException {
+
+        Configuration config = Configuration.getInstance();
+
+        config.init();
+
+        // take from demo/local/config/blueflood.conf
+        config.setProperty(CoreConfig.CASSANDRA_HOSTS, "127.0.0.1:9160");
```

Comment:
The configuration is more-or-less specific to these tests, especially the part about the `INGEST_MODE`, `QUERY_MODE`, etc being false. If the other files change, this test will still work as before. `testStartup` was the first one I wrote in this class, and was just for starters. The other tests actually test variations on the configuration.